esp/esp/program/modules/handlers/nametagmodule.py

```diff
@@ -122,6 +126,22 @@ def generatetags(self, request, tl, one, two, module, extra, prog):
 
             users = self.nametag_data(teachers, user_title)
 
+        elif idtype == 'teachermoderators':
+            user_title = "Teacher"
+            user_title2 = self.program.getModeratorTitle()
+            teacher_dict = self.program.teachers(QObjects=True)
+            teachers = ESPUser.objects.filter(teacher_dict['class_approved']).distinct()
+            moderators = ESPUser.objects.filter(teacher_dict['assigned_moderator']).distinct()
```

Comment:
Isn't this done for you if you just do `self.program.teachers()` without `QObjects=True`?

Reply:
i copied this from the previous nametag code for teachers, but i can look into it